core/src/main/java/org/p2p/core/crashlytics/helpers/CrashHttpLoggingInterceptor.kt

```diff
@@ -31,8 +31,8 @@ class CrashHttpLoggingInterceptor : Interceptor {
 
         val responseBody = response.bodyAsString()
         val responseBodySize = responseBody.length
-        if (responseBodySize < 10_000) {
-            if (!response.isSuccessful) {
+        if (responseBodySize < 7_000) {
+            if (!response.isSuccessful || responseBody.contains("error")) {
```

Comment:
sometimes response comes with 200, but with error inside, trying to identify it like this